tests/test_fork.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All rights reserved.
+
+import errno
+import pyverbs.enums as e
+from pyverbs.fork import fork_init, is_fork_initialized
+from pyverbs.pyverbs_error import PyverbsRDMAError
+
+from tests.base import PyverbsAPITestCase
+
+
+class ForkAPITest(PyverbsAPITestCase):
+    """
+    Test the API of the fork functions.
+    """
+
+    def setUp(self):
+        super().setUp()
```

Comment:
no need, the parent's setup would be called automatically...

Reply:
Done